src/pages/es/core-concepts/framework-components.mdx

```diff
@@ -11,52 +11,7 @@ Astro es compatible con una variedad de frameworks populares, incluyendo [React]
 
 ## Instalando integraciones
 
-Astro incluye integraciones opcionales de React, Preact, Svelte, Vue, SolidJS, AlpineJS y Lit. Una o varias de estas integraciones de Astro se pueden instalar y configurar en tu proyecto.
-
-Para configurar Astro para usar estos frameworks, primero, instala la integración correspondiente y cualquier peer-dependencia asociada:
-
-```bash
-npm install --save-dev @astrojs/react react react-dom
-```
-
-Luego importe y agregue la función a la lista de integraciones en `astro.config.mjs`:
-
-```js title="astro.config.mjs" ins={3} ins=/(?<!p)react\\(\\)/
-import { defineConfig } from 'astro/config';
-
-import react from '@astrojs/react';
-import preact from '@astrojs/preact';
-import svelte from '@astrojs/svelte';
-import vue from '@astrojs/vue';
-import solid from '@astrojs/solid-js';
-import lit from '@astrojs/lit';
-
-export default defineConfig({
-	integrations: [react(), preact(), svelte(), vue(), solid(), lit(), alpine()],
-});
-```
-
-⚙️ Consulte la [guía de integraciones](/es/guides/integrations-guide/) para obtener más información sobre la instalación y configuración de las integraciones de Astro.
-
-⚙️ ¿Quieres ver un ejemplo del framework de tu elección? Visite [astro.new](https://astro.new/) y seleccione la plantilla de framework correspondiente.
-
-## Usando componentes de otros frameworks
```

Comment:
I think we deleted this section by mistake, that's why the link check CI is complaining. I think it would be a great opportunity to rename the section from `## Usando componentes de otros frameworks` con `## Usando componentes de framework`. If we are doing this we also need to update the link in `src/pages/es/guides/markdown-content.mdx`